docs/tutorial/03-let-the-snake-move.md

```diff
@@ -96,7 +90,9 @@ state = %{
 }
 ```
 
-- add function to move snake and return new state:
+The snake movement is described by a shift of its body coordinates. Looking at the list that represents the snake body, we can see that this is equivalent to adding a new pair of coordinates to the beginning of the list and removing another from the end, so that the snake size is preserved.
+
```

Comment:
Should we provide an example of the shift? What do you think?
Something like:

For instance, one shift to the right is described like:

```
snake: %{body: [{9, 9}, {10, 9}, {11, 9}], size: 5, direction: {1, 0}}
```

to:
```
snake: %{body: [{10, 9}, {11, 9}, {12, 9}], size: 5, direction: {1, 0}}
```

Reply:
I agree, an example might help visualizing it :+1:

Reply:
I think it makes more sense for the direction system to be introduced in this chapter, since we're already talking about it. I'll add it.